db/migrations/sqlite/000092_add_pin_namespace.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE pins ADD COLUMN namespace VARCHAR(64);
+UPDATE pins SET namespace = "ff_system";
```

Comment:
Do you think we should we update the indexes to have `namespace` as a dimension?

I've noticed we did that elsewhere when we introduced a namespace, and I guess it should help with isolation of performance considerations between namespaces in the case of a shared DB.

Reply:
As part of answering this, I think it would be good to look at this code:
https://github.com/hyperledger/firefly/blob/eb61c4c6d21b4d85152f69ee94fd946c9b8802d4/internal/events/aggregator.go#L224-L232

If the event aggregator is to become namespace aware, then I would expect the `filter` passed into this function by the parent to include a `namespace` filter. I'm not sure if that's true today.

Reply:
Good catch, and I think this should be rolled into #855 where the aggregator becomes namespace-aware.

Reply:
Can we do that without introducing yet another migration file?

Reply:
Yes, can do. We've said migrations aren't guaranteed to be stable in mainline (only between releases).

Reply:
See https://github.com/hyperledger/firefly/pull/855/commits/f8ef5664f83fb03f71bd5951438bcab49079c20d